src/io/PipeWriter.zig

```diff
@@ -1385,11 +1386,9 @@ pub fn WindowsStreamingWriter(
 
             this.is_done = true;
             this.closed_without_reporting = false;
-            // if we are done we can call close if not we wait all the data to be flushed
-            if (this.isDone()) {
-                if (!this.owns_fd) return;
-                this.close();
-            }
+            if (this.hasPendingData()) _ = this.flush();
```

Comment:
`this.flush()` checks `is_done`. `is_done` is set to true 2 lines up.

therefore, the call to `.flush` does nothing

Reply:
close() also sets is_done to true